packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -1,6 +1,24 @@
 import {findFirstMember, insertAfterDirectives} from './utils';
 
 export default function globalAssignmentToDefaultExport({source}, {jscodeshift: j}, {printOptions = {}, appGlobalIdentifiers}) {
+  function removeFirstMember(memberExpression) {
```

Comment:
I find this logic kind of confusing. I think you should be able to do something like:

```js
j(memberExpression)
  .find(j.MemberExpression, {
    object: (node) => !j.MemberExpression.check(node)
  })
  .forEach((path) => path.replace(path.node.property));
```

To find the most deeply nested expression (`first.second`) and replace it with just the property (`second`). You should also make sure that `second` is not computed (so, bail out on something like `first[second]`)